frontend/components/Hero.js

```diff
@@ -1,6 +1,6 @@
 export default function Hero({ title, children }) {
   return (
-    <section className="hero">
+    <section className="hero px-4">
```

Comment:
Is this file even being used? Isn't that a part of the starting template?

Reply:
Yeah it's used in `src/index.js` but I don't think it needs to be a component yea